docs/tutorial/using-a-theme.md

```diff
@@ -4,42 +4,46 @@ title: Using a Theme
 
 In this tutorial, you'll learn how to use Gatsby themes by creating a new site using the official Gatsby blog theme.
 
-## Create a new site using the blog theme starter
+## Pre-requisites
 
-Creating a site using a theme starter starts the same way as using a regular Gatsby starter:
+- A Gatsby site
 
-```shell
-gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-blog-theme
-```
+> Note: This tutorial assumes you already have a Gatsby site to install your theme in. If you'd prefer to start with an entirely new site you can run `gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-blog-theme` to install a theme that comes with the starter.
 
-## Run the site
+## Installing the blog theme
 
-Creating a new site from the starter installed all of the blog theme's dependencies for you. Next, run the site and see what you have:
+Navigate to the root of your project inside your terminal and install the theme with the following command:
 
 ```shell
-cd my-blog
-gatsby develop
+  npm install gatsby-theme-blog
```

Comment:
When I install this using the starter, I get a couple of errors. First the "cannot find module React" issue with npm so I ran `npm install` again. But when I try to run `gatsby develop` after that I get this error with Node 12:

```
➜  my-blog-2 git:(master) ✗ gatsby develop

 ERROR

The above error occurred in the <StoreStateProvider> component:
    in StoreStateProvider
    in App

React will try to recreate this component tree from scratch using the error boundary you provided, App.


 ERROR

Warning: App: Error boundaries should implement getDerivedStateFromError(). In that method, return a state update to display an error message or fallback UI.


 ERROR

UNHANDLED REJECTION Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.



  Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could hap  pen for one of the following reasons:
  1. You might have mismatching versions of React and the renderer (such as React DOM)
  2. You might be breaking the Rules of Hooks
  3. You might have more than one copy of React in the same app
  See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.

  - react.development.js:1465 resolveDispatcher
    [my-blog-2]/[gatsby]/[react]/cjs/react.development.js:1465:13

  - react.development.js:1496 useState
    [my-blog-2]/[gatsby]/[react]/cjs/react.development.js:1496:20

  - context.js:21 StoreStateProvider
    [my-blog-2]/[gatsby]/[gatsby-cli]/lib/reporter/loggers/ink/context.js:21:49

  - react-reconciler.development.js:6036 renderWithHooks
    [my-blog-2]/[ink]/[react-reconciler]/cjs/react-reconciler.development.js:6036:18

  - react-reconciler.development.js:8570 mountIndeterminateComponent
    [my-blog-2]/[ink]/[react-reconciler]/cjs/react-reconciler.development.js:8570:13

  - react-reconciler.development.js:9938 beginWork$1
    [my-blog-2]/[ink]/[react-reconciler]/cjs/react-reconciler.development.js:9938:16

  - react-reconciler.development.js:11563 Object.invokeGuardedCallbackImpl
    [my-blog-2]/[ink]/[react-reconciler]/cjs/react-reconciler.development.js:11563:10

  - react-reconciler.development.js:11740 invokeGuardedCallback
    [my-blog-2]/[ink]/[react-reconciler]/cjs/react-reconciler.development.js:11740:31

  - react-reconciler.development.js:15778 beginWork$$1
    [my-blog-2]/[ink]/[react-reconciler]/cjs/react-reconciler.development.js:15778:7

  - react-reconciler.development.js:14696 performUnitOfWork
    [my-blog-2]/[ink]/[react-reconciler]/cjs/react-reconciler.development.js:14696:12
```

Reply:
I didn't look closely enough at this. Which starter were you using? Installation should still work, it's later parts of the tutorial that aren't supported by the existing 1.6 version. Would love to figure out what's happening here.